chispa/formatting/terminal_string_formatter.py

```diff
@@ -0,0 +1,21 @@
+from __future__ import annotations
+
+from chispa.formatting.formats import RESET, Format
+
+
+def format_string(input_string: str, format: Format) -> str:
```

Comment:
If it is a part of the public API, may we add also a docstring?

Reply:
Not part of the public API. At least; it's not exposed in the root `__init__.py`.